api/routes/sequence.py

```diff
@@ -149,6 +149,9 @@ async def get_sequences_by_criteria(
         else None,
     )
 
+    for sequence_group in result:
+        sequence_group.sample_id = sample_id_format(sequence_group.sample_id)
```

Comment:
I'll keep this like this so I don't get confused in my branch
```suggestion
    for sequence in result:
        sequence.sample_id = sample_id_format(sequence.sample_id)
```

Reply:
haha sorry I was trying to be good!!